extensions/healthie/validation/createTask.zod.ts

```diff
@@ -0,0 +1,69 @@
+import { isNil } from 'lodash'
+import { z } from 'zod'
+import { stringDate } from './generic.zod'
+
+const intervalTypeEnum = z.enum(['daily', 'weekly', 'once'])
+const intervalValueWeeklyEnum = z.enum([
```

Comment:
I believe this is incorrect, when `reminderIntervalType` is `weekly`, then `reminderIntervalValue` can be a comma-separated and all lower-case days of the week string(e.g "wednesday, friday"). 

How it's configured now, it will only accept 1 day of the week.

https://docs.gethealthie.com/schema/taskreminderinput.doc.html

Reply:
Thanks, I forgot about that 🙏